### ISSUE TITLE
Unable to build plugin on Debian Stretch

### ISSUE THREAD

**Issue by LDighera** [danamlund/cpu-monitor-extension-lxpanel-plugin#4]
When I issue the 'make' command, it reports:

`/root/Development/cpu-monitor-extension-lxpanel-plugin (7) # make

./check_required.sh
gcc -O2 -Wall `pkg-config --cflags glib-2.0 gtk+-2.0` \
    -shared -fPIC cpuda.c -o cpuda.so \
    `pkg-config --libs glib-2.0 gtk+-2.0`
cpuda.c: In function ‘redraw_pixmap’:
cpuda.c:113:3: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
    if (c->pixmap == NULL)
    ^~
cpuda.c:117:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
    gdk_draw_rectangle(c->pixmap, c->background_gc, TRUE, 0, 0, c->pixmap_width, c->pixmap_height);
    ^~~~~~~~~~~~~~~~~~ [indenting in this editor is munged: this ''^' is pointing to the '>' proceeding 'pixmap_height]' `

Platform information:

Debian Stretch (testing) system:
	Operating System:	GNU/Linux
	Kernel Name:		Linux
	Kernel Release:		4.9.0-2-amd64
	Kernel Version:		#1 SMP Debian 4.9.18-1 (2017-03-30)
	Machine Hardware Name:	x86_64
	Processor Type:		Intel Celeron N3160
	Hardware Platform:	Udoo X86
	Network Node Hostname:	udoox86

**Comment by danamlund** [comment]
It looks like it might just be a warning. 
You can check if there exists a 'cpuda.so' file in the folder with the Makefile. If that file exists, you should be able to 'sudo make install' to continue the installation.

**Comment by LDighera** [comment]
Thank you for your kind assistance. 

Here are the results:

`/root/Development/cpu-monitor-extension-lxpanel-plugin (7) # ls -ltr
total 96

-rw-r--r-- 1 root root  2025 May  7 09:57 README
-rw-r--r-- 1 root root   626 May  7 09:57 Makefile
drwxr-xr-x 2 root root  4096 May  7 09:57 lxpanel
-rwxr-xr-x 1 root root  2260 May  7 09:57 install.sh
-rw-r--r-- 1 root root 20865 May  7 09:57 cpuda.c
-rw-r--r-- 1 root root 18092 May  7 09:57 COPYING
-rwxr-xr-x 1 root root   623 May  7 09:57 check_required.sh
-rw-r--r-- 1 root root  1937 May  7 09:57 AUTHORS
-rwxr-xr-x 1 root root 24136 May  7 10:05 cpuda.so
-rw-r--r-- 1 root root   629 May  7 10:36 make_out.txt
/root/Development/cpu-monitor-extension-lxpanel-plugin (7) # make install
./install.sh
./install.sh: line 64: locate: command not found
'cpuda.so' -> '/usr/lib/x86_64-linux-gnu/lxpanel/plugins/cpuda.so'
`

**Comment by danamlund** [comment]
Looks like your system is missing locate, i did not know that was possible.
You can install it through 'sudo apt-get install mlocate' and then run 'sudo updatedb'.
After doing that 'make install' should finish (or at least execute a further than before).

**Comment by LDighera** [comment]
Interesting.  I found 'cpuda.so' dated "May  7 11:51" in the /usr/lib/x86_64-linux-gnu/lxpanel/plugins directory before I installed the 'locate' package (before you suggested 'mlocate'), so apparently the 'install.sh' script works without necessity of the 'locate' or 'mlocate' command.  

After running 'updatedb' then 'make install' the second time, 'cpuca.so' was overwritten as would be expected.  

`/root/Development/cpu-monitor-extension-lxpanel-plugin (7) # make install
./install.sh
'cpuda.so' -> '/usr/lib/x86_64-linux-gnu/lxpanel/plugins/cpuda.so'
`

All working fine.

Thanks again.
